geomstats/geometry/discrete_curves.py

```diff
@@ -1646,11 +1801,253 @@ def align(self, point, base_point):
         return self.aligner.align(self, point, base_point)
 
 
-class SRVTranslationReparametrizationQuotientMetric(QuotientMetric):
+class SRVReparametrizationQuotientMetric(QuotientMetric):
     """SRV quotient metric on the space of unparametrized curves.
 
     This is the class for the quotient metric induced by the SRV Metric
     on the shape space of unparametrized curves, i.e. the space of parametrized
     curves quotiented by the group of reparametrizations. In the discrete case,
     reparametrization corresponds to resampling.
     """
+
+
+class SRVRotationBundle(FiberBundle):
+    """Principal bundle of curves modulo rotations with the SRV metric.
+
+    This is the fiber bundle where the total space is the space of parameterized
+    curves equipped with the SRV metric, the action is given by rotations, and
+    the base space is the shape space of curves modulo rotations.
+
+    Parameters
+    ----------
+    total_space : DiscreteCurvesStartingAtOrigin
+        Space of discrete curves starting at the origin
+    """
+
+    def __init__(self, total_space):
+        super().__init__(total_space=total_space)
+
+    @staticmethod
+    def _transpose(point):
+        """Transpose discrete curve starting at origin."""
+        return gs.moveaxis(point, -1, -2)
+
+    def _rotate(self, point, rotation):
+        """Rotate discrete curve starting at origin."""
+        return self._transpose(gs.matmul(rotation, self._transpose(point)))
+
+    def align(self, point, base_point, return_rotation=False):
+        """Align point to base point.
+
+        Find optimal rotation of curve with respect to base curve.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
+            Discrete curve to align.
+        base_point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
+            Reference discrete curve.
+        return_rotation : boolean
+            If true, returns the optimal rotation used for the alignment.
+            Optional, default : False.
+
+        Returns
+        -------
+        aligned : array-like, shape=[..., k_sampling_points - 1, ambient_dim
+            Curve optimally rotated with respect to reference curve.
+        """
+        k_sampling_points = self.total_space.k_sampling_points
+        srv_transform = SRVTransform(
+            self.total_space.ambient_manifold,
+            k_sampling_points,
+        )
```

Comment:
If I'm not missing out anything, this diffeo is available through:

```python
srv_transform = self.total_space.quotient.metric.diffeo
```

a bit long, but probably more consistent.